### PR TITLE
feat: make v2 line map work at heath st

### PR DESCRIPTION
**Pull request by handorff**
**Asana task**: [[E-Ink feature request] GL line map update for terminals](https://app.asana.com/0/1185117109217413/1200881723138823)

This PR also stops generating alerts as candidates (since they're not ready to be displayed to riders, and Heath St screens are supposed to be installed Monday).

It also includes a couple of unrelated formatting changes to audio and normal departures, sorry! We don't actually have any checks in CI for front-end stuff, so there's no enforced standard (but maybe there should be).

### CODE REVIEW

File: lib/screens/v2/widget_instance/line_map.ex [hunk 1]
```diff
@@ -52,29 +81,98 @@ defmodule Screens.V2.WidgetInstance.LineMap do
...
+        reverse_stops
+        |> Enum.reverse()
+        |> Enum.drop(1)
+        |> Enum.with_index()
+        |> Enum.map(fn {%Screens.Stops.Stop{name: label}, i} ->
+          %{
+            label: label,
+            downstream: false,
+            current: false,
+            terminal: i == length(reverse_stops) - 2
```
Comment (jzimbel-mbta) [inline lib/screens/v2/widget_instance/line_map.ex:112]: Could you explain why it's `length(reverse_stops) - *2*`? (Also would this ever happen? I think all of our lines are long enough that you wouldn't see the whole length from one terminus to the other on the e-ink line map)

File: lib/screens/v2/widget_instance/line_map.ex [hunk 2]
```diff
@@ -3,25 +3,35 @@ defmodule Screens.V2.WidgetInstance.LineMap do
 
   alias Screens.Config.{Screen, V2}
   alias Screens.Predictions.Prediction
+  alias Screens.Trips.Trip
```
Comment (jzimbel-mbta) [inline lib/screens/v2/widget_instance/line_map.ex:6]: Doesn't look like `Trip` is used anywhere.

File: test/screens/v2/widget_instance/line_map_test.exs [hunk 3]
```diff
@@ -4,6 +4,7 @@ defmodule Screens.V2.WidgetInstance.LineMapTest do
   alias Screens.Predictions.Prediction
   alias Screens.Schedules.Schedule
   alias Screens.Stops.Stop
+  alias Screens.Trips.Trip
```
Comment (jzimbel-mbta) [inline test/screens/v2/widget_instance/line_map_test.exs:7]: Doesn't look like `Trip` is used anywhere.

Comment (handorff) [reply test/screens/v2/widget_instance/line_map_test.exs:7]: I think it is (e.g. here: https://github.com/mbta/screens/pull/1070/files#diff-57482df2b19067549da87f0f7d0d88f13aa39dc250a41473aa1ade5103c2b610R110).

**Pull request merged**